blueocean-dashboard/src/main/js/creation/TempUtils.js

```diff
@@ -0,0 +1,18 @@
+
+function cleanSlashes(url, trailingSlash = true) {
```

Comment:
If I had a million dollars, 2 functions at the same time. 

<img src='https://i.ytimg.com/vi/A939QRRSNV4/maxresdefault.jpg'>